src/test/acceptance/features/send-code.feature

```diff
@@ -8,6 +8,8 @@ Feature: We're going to send you a code
 
   Scenario: Text and Email are displayed
     Then Text and Email options are displayed on the send code page
+    And The change-text link points to the phone number page
```

Comment:
Why the need for dashes in `change-text` and `change-email`?

Reply:
No reason - that was the id I used. I'll remove it from the step name.